rapids-cmake/cython/create_modules.cmake

```diff
@@ -117,8 +117,15 @@ function(rapids_cython_create_modules)
     endif()
     install(TARGETS ${cython_module} DESTINATION ${_RAPIDS_CYTHON_INSTALL_DIR})
 
-    # Default the INSTALL_RPATH for all modules to $ORIGIN.
-    set_target_properties(${cython_module} PROPERTIES INSTALL_RPATH "\$ORIGIN")
+    # Default the INSTALL_RPATH for all modules to $ORIGIN. We also append any lib dirs specified in
```

Comment:
We should add a notes to the docs of this fuction that cover what install rpaths will be set.